containers/Makefile

```diff
@@ -39,6 +39,9 @@ build/concent-api/concent-api.tar: $(CONCENT_DIR)
 	mkdir --parents $(dir $@)
 	package-builder/build-concent-api.sh "$(CONCENT_DIR)" "build/concent-api"
 
+build/nginx-proxy/concent-api-assets.tar: $(CONCENT_DIR)
+	mkdir --parents $(dir $@)
+	package-builder/build-concent-api-assets.sh "$(CONCENT_DIR)" "../../nginx-proxy"
```

Comment:
`../../nginx-proxy` here is relative to the `build/concent-api-assets/concent_api` path we `cd` into inside the script, which, in turn is relative to the directory of the `Makefile`. This is different from other paths in the `Makefile` and very confusing for someone reading the script and will surely be a source of future bugs.

You should always assume that the paths are relative to script's working directory. If you need something relative to another directory, just use the absolute path.

In bash, use `$(realpath "dir")` to get the absolute path. In make use `$(abspath dir)`